service/controller/v24/key/key.go

```diff
@@ -178,13 +177,6 @@ func CloudFormationHostPostTemplates() []string {
 	}
 }
 
-func CloudFormationHostPreTemplates() []string {
-	return []string{
-		cpi.IAMRoles,
-		cpi.Main,
-	}
-}
```

Comment:
So this here is cool. I wanted to get this out of the key package for a long time.